ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -662,7 +663,8 @@ has not been specified for this external model and machine combination:
       sysdir="$sysbasedir/gfs.${yyyymmdd}/${hh}/atmos"
       ;;
     "HERA")
-      sysdir="$sysbasedir/gfs.${yyyymmdd}/${hh}/atmos"
+      sysdir="$sysbasedir"
```

Comment:
Again, these changes will only work for the staged GFS data that were copied from JET. However, the directory structure of the real-time GFS data on HERA are not using this format.